lib/plugins/features/movies/controller.js

```diff
@@ -3,6 +3,9 @@
 const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
-  return new Movie().save(payload)
+  return new Movie().save({
+    name: payload.title,
```

Comment:
we also dont want to stop saving `title` yet either. because after this change is deployed, there will be some rows with `name` and others with `title` and that gets a bit confusing to deal with (though not impossible). lets talk irl about the best way to tackle this